src/styles/blocks/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
```

```suggestion
  &--4 :nth-child(-n + 4),
  ... {
```
Consider adding other modifiers here